CIP-0001/README.md

```diff
@@ -240,6 +240,8 @@ This must be subdivided into two sub-sections:
 
     This sub-section must define a list of criteria by which the proposal can become active. Criteria must relate to observable metrics or deliverables and be reviewed by editors and project maintainers when applicable. For example: "The changes to the ledger rules are implemented and deployed on Cardano mainnet by a majority of the network", or "The following key projects have implemented support for this standard".
 
+    For proposals under core categories (Ledger, Plutus, Network, Consensus), _'Acceptance Criteria'_ must include reference to multiple node implementations.
```

Comment:
Until just now, I wasn't really sure myself what a `core` category was! 😅 Maybe we should enumerate these in the _Categories_ section of CIP-0001 (which CIP-9999 also refers to)?

I'll push a commit to CIP-0001 into this that I think simply conveys which ones are `core` so it's apparent earlier in CIP-0001 (by adding it explicitly to the Category table headings); stay tuned...